tests/cases/fourslash/importNameCodeFixJsEnding.ts

```diff
@@ -0,0 +1,17 @@
+/// <reference path="fourslash.ts"/>
+
+// @module: commonjs
+
+// @Filename: /node_modules/lit/package.json
+//// { "name": "lit", "version": "1.0.0" }
+
+// @Filename: /node_modules/lit/index.d.ts
+//// import "./decorators";
+
+// @Filename: /node_modules/lit/decorators.d.ts
+//// export declare function customElement(name: string): any;
+
+// @Filename: /a.ts
+//// customElement/**/
+
+verify.importFixModuleSpecifiers("", ["lit/decorators.js"], { importModuleSpecifierEnding: "js" })
```

Comment:
should there be a test for .cjs/.mjs as well, since you mentioned that they were broken before?

Reply:
also, are there already tests for filename/package conflicts? I assume yes.

Reply:
There are .cjs tests. Filename/package conflicts aren’t related to this change, I think.